spark3/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -474,15 +475,28 @@ public static boolean isLocalityEnabled(FileIO io, String location, CaseInsensit
     return false;
   }
 
-  public static boolean isVectorizationEnabled(Map<String, String> properties, CaseInsensitiveStringMap readOptions) {
+  public static boolean isVectorizationEnabled(FileFormat fileFormat,
```

Comment:
How important is it to keep this API? Seems internal to me.

Reply:
Seems internal to me too. I would be +1 on changing it. I miss Scala's package scope access modifiers in such cases.

Reply:
Oh, yeah!